src/main/java/org/cascadebot/cascadebot/commands/moderation/PurgeAttachmentsSubCommand.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.commands.moderation;
+
+import net.dv8tion.jda.api.entities.Member;
+import org.cascadebot.cascadebot.commandmeta.*;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.utils.PurgeUtils;
+
+
+public class PurgeAttachmentsSubCommand implements ISubCommand {
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        if (context.getArgs().length == 0) {
```

Comment:
This doesn't check whether the args is of length 2 like it should

Reply:
That isn't neccesarily? 
`;purge attachments 1`

Reply:
Then why does the purge method have `context.getArgAsInteger(1), context.getArg(0)`? Should it be `context.getArgAsInteger(0), null`?

Reply:
There is contain/user that needed an additional argument specifying more. `;purge contains "example" 10`
Whilst **ALL** of the other command need 1 argument, amount.